code-scanning/flawfinder.yml

```
@@ -0,0 +1,38 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: flawfinder
+
+on:
+  push:
+    branches: [ $default-branch, $protected-branches ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ $default-branch ]
+  schedule:
+    - cron: $cron-weekly
+
+jobs:
+  flawfinder:
+    name: Flawfinder
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: flawfinder_scan
+        uses: david-a-wheeler/flawfinder@2.0.19-alpha1
```

I'm afraid we need a full SHA here rather than a tag. :-)

```suggestion
        uses: david-a-wheeler/flawfinder@51c988dc47885f525022b4d393ff4479024499f7
```

I took the SHA from the most recent stable version (rather than the one marked 'alpha').

---

@nickfyson thanks for reviewing! The commit 51c988dc47885f525022b4d393ff4479024499f7 was submitted on Jun 24, there are couple changes related to action were checked in after that commit.
Should we use the latest commit hash 8e4a779ad59dbfaee5da586aa9210853b701959c?

---

If that's the best choice, then happy to go with that instead! 🙂👍

---

thanks for confirming! Updated the action string using latest commit hash.
pls lets know if any other issues need to be fixed. 👍